<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementForm</class>
 <widget class="QWidget" name="MeasurementForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,9" columnstretch="1,9">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="measureToolButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icons/Icons.qrc">
         <normaloff>:/control_play_blue.png</normaloff>:/control_play_blue.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveToolButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icons/Icons.qrc">
         <normaloff>:/disk_faded.png</normaloff>:/disk_faded.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="3">
      <widget class="PlotForm" name="mPlotWidget_16" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="PlotForm" name="mPlotWidget_13" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="PlotForm" name="mPlotWidget_09" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="PlotForm" name="mPlotWidget_10" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="PlotForm" name="mPlotWidget_05" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="PlotForm" name="mPlotWidget_06" native="true"/>
     </item>
     <item row="2" column="3">
      <widget class="PlotForm" name="mPlotWidget_12" native="true"/>
     </item>
     <item row="2" column="2">
      <widget class="PlotForm" name="mPlotWidget_11" native="true"/>
     </item>
     <item row="3" column="2">
      <widget class="PlotForm" name="mPlotWidget_15" native="true"/>
     </item>
     <item row="3" column="1">
      <widget class="PlotForm" name="mPlotWidget_14" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="PlotForm" name="mPlotWidget_07" native="true"/>
     </item>
     <item row="1" column="3">
      <widget class="PlotForm" name="mPlotWidget_08" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="PlotForm" name="mPlotWidget_01" native="true">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PlotForm" name="mPlotWidget_02" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="PlotForm" name="mPlotWidget_03" native="true"/>
     </item>
     <item row="0" column="3">
      <widget class="PlotForm" name="mPlotWidget_04" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="hHeaderWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Analyzer State 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Analyzer State 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Analyzer State 3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Analyzer State 4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="vHeaderWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Polarizer State 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Polarizer State 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Polarizer State 3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Polarizer State 4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotForm</class>
   <extends>QWidget</extends>
   <header location="global">plotform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
